colin-api/src/colin_api/models/filing.py

```diff
@@ -683,6 +683,8 @@ def get_filing(cls, filing: Filing, con=None, year: int = None) -> Dict:
                     filing.body['business']['legalType'] = Business.TypeCodes.BC_COMP.value
                 elif filing_event_info['filing_type_code'] == 'NOALE':
                     filing.body['business']['legalType'] = Business.TypeCodes.BCOMP.value
+                elif filing_event_info['filing_type_code'] == 'NOALA':
+                    filing.body['business']['legalType'] = filing.business.corp_type
```

Comment:
does this mean a filing with a NOALA isn't changing the legal type? This will set it to the current business legal type of the business. The output will look okay unless a company has multiple alterations in their history or something like that (and therefore the legal type would be incorrect and silently update lear incorrectly as well if the legal_updater was getting this filing). This is probably very rare but still possible

Reply:
After talking with @kialj876 about this and looking into the colin database structure, we are unable to determine what the legal type is at the time of a NOALA filing.  As per Kial's previous comment, this means there are scenarios where if a business has changed business types multiple times, we would not be able to pull over the right filing with the right legal type.

Currently, when the legal updater tries to pull over a NOALA filing it fails when trying to access the legal type.  Should we just keep letting it fail?  Or is there some way we can find the legal type of a business at the time of a particular NOALA filing?

Reply:
that sounds wrong. I did reach out to the DBA team and they confirm we always know what the corp_type is.